### PR TITLE
Ajust video tag for video config

### PR DESCRIPTION
**Pull request by LuxoftAKutsan**

### CODE REVIEW

File: app/model/sdl/Abstract/Model.js [hunk 1]
```diff
@@ -168,17 +169,27 @@ SDL.SDLModel = Em.Object.extend({
           return;
         }
 
+        var active_app_id = SDL.SDLController.model.appID
+        var app_model = SDL.SDLController.getApplicationModel(active_app_id)
+            
+        var video_width = SDL.SDLController.getApplicationModel(active_app_id).VideoConfigWidth
+        var video_height = SDL.SDLController.getApplicationModel(active_app_id).VideoConfigHeight
+
+        var marging_left = ((800 - video_width) / 2)
```
Comment (YarikMamykin) [inline app/model/sdl/Abstract/Model.js]: Magic numbers?

File: ffw/NavigationRPC.js [hunk 2]
```diff
@@ -416,6 +416,10 @@ FFW.Navigation = FFW.RPCObserver.create(
           }
           case 'Navigation.SetVideoConfig':
           {
+ 
+              var app_model = SDL.SDLController.getApplicationModel(request.params.appID)
```
Comment (YarikMamykin) [inline ffw/NavigationRPC.js]: Some check for app_model validity should be great here))

**Review by YarikMamykin — CHANGES_REQUESTED** [review]

File: app/model/sdl/Abstract/Model.js [hunk 3]
```diff
@@ -168,17 +169,27 @@ SDL.SDLModel = Em.Object.extend({
           return;
         }
 
+        var active_app_id = SDL.SDLController.model.appID
+        var app_model = SDL.SDLController.getApplicationModel(active_app_id)
```
Comment (AKalinich-Luxoft) [inline app/model/sdl/Abstract/Model.js]: @LuxoftAKutsan suggest to use `return` if app_model` == NULL

File: app/model/sdl/Abstract/Model.js [hunk 4]
```diff
@@ -168,17 +169,27 @@ SDL.SDLModel = Em.Object.extend({
...
+            
+        var video_width = SDL.SDLController.getApplicationModel(active_app_id).VideoConfigWidth
+        var video_height = SDL.SDLController.getApplicationModel(active_app_id).VideoConfigHeight
```
Comment (AKalinich-Luxoft) [inline app/model/sdl/Abstract/Model.js]:
```suggestion
        var video_width = app_model.VideoConfigWidth;
        var video_height = app_model.VideoConfigHeight;
```

File: app/model/sdl/Abstract/Model.js (hunk 1 above)
Comment (AKalinich-Luxoft) [inline app/model/sdl/Abstract/Model.js]: @LuxoftAKutsan I think you can use something like this to get actual work area size:
```
var view = document.getElementById('baseNavigation');
const view_width = view.offsetWidth;
const view_height = view.offsetHeight;
```

And then use it in your margin calculations:
```
var margin_left = ((view_width - video_width) / 2);
var margin_top = ((view_height - video_height) / 2);
```

File: app/model/sdl/Abstract/Model.js [hunk 5]
```diff
@@ -666,7 +677,24 @@ SDL.SDLModel = Em.Object.extend({
         var sdl_stream = SDL.SDLController.getApplicationModel(
           appID
         ).navigationStream;
-
+        
+        console.log(' set params from VideoConfig');
```
Comment (AKalinich-Luxoft) [inline app/model/sdl/Abstract/Model.js]: @LuxoftAKutsan please don't use console output directly. We have a dedicated class for logging:
```
Em.Logger.log(`Set params from VideoConfig`);
```

File: app/model/sdl/Abstract/Model.js [hunk 6]
```diff
@@ -666,7 +677,24 @@ SDL.SDLModel = Em.Object.extend({
...
+        
+        var app_model = SDL.SDLController.getApplicationModel(appID)
+        console.log("playVideo app_model", app_model)
```
Comment (AKalinich-Luxoft) [inline app/model/sdl/Abstract/Model.js]: @LuxoftAKutsan same as above

File: app/model/sdl/Abstract/Model.js [hunk 7]
```diff
@@ -666,7 +677,24 @@ SDL.SDLModel = Em.Object.extend({
...
+            
+
+        var width = SDL.SDLController.getApplicationModel(appID).VideoConfigWidth
+        var height = SDL.SDLController.getApplicationModel(appID).VideoConfigHeight
```
Comment (AKalinich-Luxoft) [inline app/model/sdl/Abstract/Model.js]:
```suggestion
        var height = app_model.VideoConfigHeight;
```

File: app/model/sdl/Abstract/Model.js [hunk 8]
```diff
@@ -666,7 +677,24 @@ SDL.SDLModel = Em.Object.extend({
...
+        
+        var app_model = SDL.SDLController.getApplicationModel(appID)
+        console.log("playVideo app_model", app_model)
+            
+
+        var width = SDL.SDLController.getApplicationModel(appID).VideoConfigWidth
+        var height = SDL.SDLController.getApplicationModel(appID).VideoConfigHeight 
+        
+        var marging_left = ((800 - width) / 2) 
+        var marging_top = ((400 - height) / 2)
```
Comment (AKalinich-Luxoft) [inline app/model/sdl/Abstract/Model.js]: @LuxoftAKutsan please use the same approach with width and height as I mentioned above

**Review by AKalinich-Luxoft — CHANGES_REQUESTED** [review]

File: app/model/sdl/Abstract/Model.js (hunk 3 above)
Comment (LuxoftAKutsan) [reply app/model/sdl/Abstract/Model.js]: 72bd13d

File: app/model/sdl/Abstract/Model.js (hunk 1 above)
Comment (LuxoftAKutsan) [reply app/model/sdl/Abstract/Model.js]: 72bd13d

File: app/model/sdl/Abstract/Model.js (hunk 5 above)
Comment (LuxoftAKutsan) [reply app/model/sdl/Abstract/Model.js]: 72bd13d

File: app/model/sdl/Abstract/Model.js (hunk 8 above)
Comment (LuxoftAKutsan) [reply app/model/sdl/Abstract/Model.js]: 72bd13d

File: app/model/sdl/Abstract/Model.js (hunk 6 above)
Comment (LuxoftAKutsan) [reply app/model/sdl/Abstract/Model.js]: 72bd13d

File: ffw/NavigationRPC.js (hunk 2 above)
Comment (LuxoftAKutsan) [reply ffw/NavigationRPC.js]: 72bd13d

**Review by YarikMamykin — APPROVED** [review]

**Pull request merged**